### PR TITLE
Use camel-case args for countsFromAbundance option

### PR DESCRIPTION
**Pull request by wm75**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by wm75** [comment]
Hmm, I fixed all linting errors in the R code except one: `tools/tximport/tximport.R:78:13 Variable and function name style should be snake_case.`
I don't see how I can fix that remaining one though without a deeper understanding of the tool. All I really wanted to do here is fix a user-reported bug in the tool, not familiarize myself with tx-import much.

@ieguinoa @nsoranzo maybe?

**Comment by nsoranzo** [comment]
@wm75 Sorry for the late reply, I've added a commit to skip the linting on this line as we already had to do for `tools/deseq2/get_deseq_dataset.R` some time ago.

File: tools/tximport/tximport.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
-<tool id="tximport" name="tximport" version="0.1">
+<tool id="tximport" name="tximport" version="1.20.0">
```
Comment (nsoranzo) [inline tools/tximport/tximport.xml:1]: Use a version macro?

File: tools/tximport/tximport.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
     <description>Summarize transcript-level estimates for gene-level analysis</description>
     <requirements>
-        <requirement type="package">bioconductor-tximport</requirement>
+        <requirement type="package" version="1.20.0">bioconductor-tximport</requirement>
         <requirement type="package" version="1.34.1">bioconductor-genomicfeatures</requirement>
```
Comment (nsoranzo) [inline tools/tximport/tximport.xml:5]:
```suggestion
        <requirement type="package" version="1.44.0">bioconductor-genomicfeatures</requirement>
```

File: tools/tximport/tximport.xml [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
         <requirement type="package" version="1.20.2">r-getopt</requirement>
```
Comment (nsoranzo) [inline tools/tximport/tximport.xml:6]:
```suggestion
        <requirement type="package" version="1.20.3">r-getopt</requirement>
```

File: tools/tximport/tximport.xml [hunk 4]
```diff
@@ -1,7 +1,7 @@
-<tool id="tximport" name="tximport" version="0.1">
+<tool id="tximport" name="tximport" version="1.20.0">
     <description>Summarize transcript-level estimates for gene-level analysis</description>
     <requirements>
-        <requirement type="package">bioconductor-tximport</requirement>
+        <requirement type="package" version="1.20.0">bioconductor-tximport</requirement>
```
Comment (nsoranzo) [inline tools/tximport/tximport.xml:4]:
```suggestion
        <requirement type="package" version="@TOOL_VERSION@">bioconductor-tximport</requirement>
```

**Comment by wm75** [comment]
Thanks @nsoranzo for helping with turning tests green here :)

**Review by nsoranzo — APPROVED** [review]

**Pull request merged**